values.yaml

```diff
@@ -104,15 +99,13 @@ interpolation:
     cpu: 0.1
     # cannot use a hyphen to match the k8s param due to https://github.com/helm/helm/issues/2192
     ephemeral_storage: 5Gi
-  annotations: {}
-  nodeSelector: {}
-  pvc: {}
-#    create: true
-#    name: interpolation-pvc
-#    storageClass: aws-efs
-#    accessModes: ReadWriteMany
-#    storage: 10Gi
-#    annotations: {}
+  pvc:
```

Comment:
keep original. overriding values will be in another file that should not be committed to this repo